### PR TITLE
Fix issue with Use of Force Complaints not showing up

### LINKED ISSUE #9: Use of Force Complaint Sustained Not Showing
Overview:
---

It's still not showing the percent of use of force complaints sustained even though there’s a 0% for `calc_use_of_force_complaints_sustained`. It should only hide this when it’s blank, not when it’s a zero.

Details:
---

> Info about the Environment:

* **URL**: https://staging.policescorecard.org/mo/police-department/st-louis

Acceptance Criteria:
---

> Issue will be considered complete when the following criteria are met:

- [ ] Use of Force Complaints should show Complaints Sustained if exists and is a number ( including zero )
- [ ] Use of Force Complaints should hide Complaints Sustained if value is null or undefined

Documentation:
---

> Screenshots, Attachments, Linked GitHub Issues, etc

image

Dev & QA Notes:
---

> Any additional notes from developers or QA required to complete this issue.

### PR DESCRIPTION
**Pull request by manifestinteractive**
Overview:
---

User of Force Complaints Sustained was not showing up, even though it had a value of zero.

Reviewer:
---

> Where should the reviewer start? How to Test? Background Context? etc ( required )

Conditional logic here was treating a `0` as `false`.  Only time we are supposed to hide this is if the value is empty ( not zero ).

Checklist:
---

> I have tested each of the following, and they work as expected: ( required )

- [X] Meets [Contributing Guide](https://github.com/campaignzero/police-scorecard/blob/develop/.github/CONTRIBUTING.md) Requirements
- [X] Pulled in the Latest Code from the `develop` branch
- [X] Works on a Desktop / Laptop Device
- [X] Works on a Mobile Device
- [X] `npm test` Does Not Generate Errors

Documentation:
---

> Screenshots, Attachments, Linked GitHub Issues, etc ( optional )

Fixes #9 

screenshot 2020-09-09 at 2 23 19 AM

#### What GIF best describes this PR or how it makes you feel?

> Drag & Drop Something Fun Here ( optional )

yup

### CODE REVIEW

File: resources/views/components/partial/police-accountability/chart-use-of-force.blade.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <p>
         {{ num($scorecard['police_accountability']['use_of_force_complaints_reported']) }} Reported
 
-        @if($scorecard['report']['percent_use_of_force_complaints_sustained'])
+        @if(isset($scorecard['report']['percent_use_of_force_complaints_sustained']))
```
Comment (MasterInTraining23) [inline resources/views/components/partial/police-accountability/chart-use-of-force.blade.php:14]: Extract into variable since it's used many times.

**Review by MasterInTraining23 — CHANGES_REQUESTED** [review]

File: resources/views/components/partial/police-accountability/chart-use-of-force.blade.php (hunk 1 above)
Comment (manifestinteractive) [reply resources/views/components/partial/police-accountability/chart-use-of-force.blade.php:14]: It already is a variable `$scorecard` is loaded in the route and that same variable is passed through all the partial templates.  That variable is just a multidimensional array that contains all the data needed. There are several hundred places in this project we would need to make out custom template variables if we went this route, so do not make sense to do this for just this PR.

Comment (MasterInTraining23) [reply resources/views/components/partial/police-accountability/chart-use-of-force.blade.php:14]: "$scorecard['report']['percent_use_of_force_complaints_sustained']" this variable is used multiple times. The number of times we index into the map outweighs the memory alloc and lack of readability.

Comment (manifestinteractive) [reply resources/views/components/partial/police-accountability/chart-use-of-force.blade.php:14]: right, my point though is that we are doing this ALL OVER this project, just just here.  If we want to tweak this, which I am fine with, we should make this a dedicated ticket and redo ALL instances of where this happens, not just this one line in this PR where it happens to be happening.

If there are performance gains to streamline this, it should be done site wide as a dedicated ticket, as just doing it for this one instance kind of does not make sense IMO when the rest of the code base has the exact same issues.

Totally fine knocking this out on my end, but just don't think it should block this specific PR from going out.

Comment (MasterInTraining23) [reply resources/views/components/partial/police-accountability/chart-use-of-force.blade.php:14]: Good point. Let's refactor in a separate change.

**Review by MasterInTraining23 — APPROVED** [review]

**Pull request merged**